### PR TITLE
new packages: tetgen

### PR DESCRIPTION
**Pull request by 3rav**
Base on: https://aur.archlinux.org/packages/tetgen/

### CODE REVIEW

File: mingw-w64-tetgen/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+pkgdesc='A Quality Tetrahedral Mesh Generator and a 3D Delaunay Triangulator'
+arch=('any')
+makedepends=("${MINGW_PACKAGE_PREFIX}-ninja"
+             "${MINGW_PACKAGE_PREFIX}-cmake")
+url='http://wias-berlin.de/software/index.jsp?id=TetGen&lang=1'
+license=('AGPL3')
+# You must accept the licensing terms before downloading the source archive:
+#   http://wias-berlin.de/software/tetgen/download2.jsp
+# Place the downloaded archive in $SRCDEST of your /etc/makepkg.conf
+source=("local://${pkgname}${pkgver}.tar.gz")
```
Comment (lazka) [inline mingw-w64-tetgen/PKGBUILD:17]: Use `www.tetgen.org/1.5/src/tetgen1.6.0.tar.gz`

File: mingw-w64-tetgen/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+options=('staticlibs')
+
+build() {
+  [[ -d "${srcdir}/build-${MINGW_CHOST}" ]] && rm -rf "${srcdir}/build-${MINGW_CHOST}"
+  mkdir -p "${srcdir}/build-${MINGW_CHOST}" && cd "${srcdir}/build-${MINGW_CHOST}"
+
+  MSYS2_ARG_CONV_EXCL="-DCMAKE_INSTALL_PREFIX=" \
+    ${MINGW_PREFIX}/bin/cmake \
+        -GNinja \
+        -DCMAKE_INSTALL_PREFIX=/usr \
```
Comment (lazka) [inline mingw-w64-tetgen/PKGBUILD:28]: wrong prefix

File: mingw-w64-tetgen/PKGBUILD [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+        -DCMAKE_BUILD_TYPE=Release \
+	-DCMAKE_EXE_LINKER_FLAGS="-static" \
+	../"${pkgname}${pkgver}"
+  ninja
+}
+
+package() {
+  cd "${srcdir}/build-${MINGW_CHOST}"
+  install -m 755 -D -t "${pkgdir}${MINGW_PREFIX}/bin" tetgen
```
Comment (lazka) [inline mingw-w64-tetgen/PKGBUILD:37]: Does "ninja install" not work like everywhere else?

Comment (3rav) [reply mingw-w64-tetgen/PKGBUILD]: With "DESTDIR=${pkgdir} cmake --build . --target install":

```
[6/6] Linking CXX executable tetgen.exe
==> Starting package()...
ninja: error: unknown target 'install'
==> ERROR: A failure occurred in package().
```

**Pull request merged**